deployment/scripts/cleanup.sh

```
@@ -0,0 +1,250 @@
+    echo "3. No old releases to remove"
+    echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+fi
+
+# 4. Docker cleanup
+echo ""
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo "4. Docker resources cleanup"
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+
+# Get disk usage before cleanup
+DOCKER_BEFORE=$(docker system df --format "{{.Size}}" | head -n 1 || echo "0B")
```

Shellcheck (0.11.0)

[warning] DOCKER_BEFORE appears unused. Verify use (or export if used externally).

(SC2034)